### PR TITLE
Comments page and functionalities

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
# Comments page and functionalities

## Preview Link: [LUCENT MEDIA](https://deploy-preview-6--lucent-media.netlify.app/)

Used Tailwind CSS for styling the components.

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
Well understandable code with all the segregation made using comments
Good work 
seems to to merge

File: src/components/comment-modal/CommentModal.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            onClick={() => {
+              dispatch(editComment({ postId, commentId, text }));
+              setIsEditComment(false);
+            }}
+          >
+            EDIT
+          </button>
+          <button
+            className="px-4 py-1 rounded text-green-50 bg-green-600 hover:bg-green-700"
+            onClick={() => {
```
Comment (bhtibrewal) [inline src/components/comment-modal/CommentModal.jsx]: Suggestion: Move this into functions as it makes it easy to understand what will happen onClick

File: src/components/routes/Routing.jsx [hunk 2]
```diff
@@ -39,6 +41,14 @@ const Routing = () => {
           </RequireAuth>
         }
       />
+      <Route
+        path="/comment"
+        element={
+          <RequireAuth>
```
Comment (bhtibrewal) [inline src/components/routes/Routing.jsx]: Suggestion: you can make RequireAuth with Outlet and wrap all the private routes in one single ReuireAuth

File: src/redux-management/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  likePost,
+} from "./slices/postSlice/postServices/likeServices";
+export {
+  createNewPost,
+  deletePost,
+  editPost,
+  getAllPosts,
+  getPostById,
+  getPostsByUsername,
+} from "./slices/postSlice/postServices/postServices";
```
Comment (bhtibrewal) [inline src/redux-management/index.js:23]: Appreciation: good work on exporting all fuctions from one page

**Review by bhtibrewal — APPROVED** [review]
Written few small suggestions. Else good work on the code , nicely done👍

**Pull request merged**